packages/hyperion-autologging/src/ALNetworkPublisher.ts

```diff
@@ -26,7 +26,7 @@ type ALNetworkEvent = ALTimedEvent & Readonly<{
  */
 type RequestInfo = {
   method: string;
-  url: string | URL; // TODO: should we always send on of the types?
+  url: string;
```

Comment:
since usually parsing a url is tricky, how about we add an optional `parsedUrl` field and keep the `URL` value in that one as well?